configuring.tex

```diff
@@ -1,424 +1,185 @@
+  \includegraphics[width=0.7\linewidth]{images/ss-m65config-1.png}
 \end{center}
 
-To avoid accidental loss of data, you must type \screentext{DELETE EVERYTHING} in capitals and press \specialkey{RETURN}. Alternatively, switch the MEGA65 off and on to abort this process without causing damage to your data.
+The MEGA65 supports the Commodore 1351 mouse,\index{Commodore 1351 mouse} the Commodore Amiga mouse,\index{Commodore Amiga mouse} or modern equivalents such as a USB mouse connected with a \href{https://retrohax.net/shop/amiga/mouster/}{mouSTer} adapter.\index{mouSTer adapter} The port must be set to the correct mouse type, where {\bf normal} refers to the 1351 mouse. If an Amiga mouse is connected while the port is in the {\bf normal} mode, it may interfere with the behavior of the keyboard.
 
-It is also possible to attempt a recovery from a lost Master Boot Record error (also known as the Boot Sector), by typing \screentext{FIX MBR} instead.
+{\bf Amiga mouse detection} controls how the MEGA65 interprets Amiga mouse signals and converts them to equivalent 1351 signals when the port is in Amiga mouse mode. Some joysticks experience interference on a port with Amiga mouse mode enabled. If you notice any such interference, change this setting to {\bf conservative,} otherwise leave it set to {\bf aggressive.}
 
-The aim here is to have a correctly formatted SD card with all of the essential files stored on it so the MEGA65 can properly boot.
-When switching on, the MEGA65 will search for, and boot using these files:
-\begin{itemize}
-\item {\tt FREEZER.M65} (Freeze Menu program)
-\item {\tt AUDIOMIX.M65} (Freeze Menu audio mixer utility)
-\item {\tt C64THUMB.M65} (C64 thumbnail image used in freezer)
-\item {\tt C65THUMB.M65} (C65 thumbnail image used in freezer)
-\item {\tt MEGA65.ROM}   (128KB ROM file)
-\item {\tt MEGA65.D81} (Disk image)
-\end{itemize}
+\subsection{Chipset}
+\index{Configuration!Boot Disk}
 
-Straight out of the box, the MEGA65 will only have one SD card installed, accessible via the trap-door under the case. This SD card contains all of the essential files needed to properly boot up.
-If an external microSD card is also detected during boot up, the MEGA65 will give it higher priority, and will try to boot from it instead.
-This means that the external microSD card needs to have the essential files on it, otherwise the MEGA65 will not boot up properly and will fall back to loading the OpenROM, which does not support all MEGA65 features.
-In general, if your MEGA65 cannot boot properly and falls back to OpenROM, your boot-up screen will look similar to this:
+The {\bf Chipset} page configures several features, including the Real-Time Clock.
 
 \begin{center}
-\includegraphics[trim=0 8cm 0 0,clip,width=0.7\linewidth]{images/mega65_OpenROM_boot_noSD.png}
+  \includegraphics[width=0.7\linewidth]{images/ss-m65config-2.png}
 \end{center}
```

Comment:
Do you know how to reproduce an openrom default boot? If I nix mega65.rom I just get "could not find rom mega65.rom" in hyppo and it loops. Same if I nix all system files. Is this still a feature?

Reply:
Yeah, this is a funny one. For a long time, the way the docs described it wasn't the way it actually triggered.

For a long time, it only triggered when you took all sd-cards out and re-booted.

I recently made a tweak to hyppo (merged into 'development' core branch) so that in addition to the above case, it would also bail out to open roms if no MEGA65.ROM was found:
- https://github.com/MEGA65/mega65-core/commit/ea3508ea50298f879ed1a5ae527ab3abd2545251

Alas, this behaviour only works in PAL, due to NTSC version choking on some raster issue that FeralChild mentioned in this issue he raised on the core:
- https://github.com/MEGA65/mega65-core/issues/362

So for ntsc, open rom usually ends up stalled on a blue-screen (which I think you can break out of with RUNSTOP-RESTORE to get to the `READY.` prompt if needed for now).

Reply:
It sounds like I would need the latest dev core in slot 0 for this to work, yeah? On my machine the boot process stops with an error message, and I have 683-cartflash in slot 0. I'll see if I can get this behavior working and figure out where it should be mentioned.